slides/section_4_collaborative_soft_dev.md

```diff
@@ -305,7 +305,7 @@ poetry --version  # check we have access to the poetry executable
   git clone <our_repo>
   python -m venv venv
   . venv/bin/activate
-  pip install -r requirements.txt
+  pip install -r requirements.txt --editable .
```

Comment:
```suggestion
  pip install -r requirements.txt
```

Reply:
I think in this case it should stay here. After episode `12-virtual-environments` all installations of our project should be done in editable mode.